editor/src/messages/portfolio/document/node_graph/node_graph_message_handler/node_properties.rs

```diff
@@ -161,12 +153,11 @@ fn number_widget(document_node: &DocumentNode, node_id: NodeId, index: usize, na
 /// Properties for the input node, with information describing how frames work and a refresh button
 pub fn input_properties(_document_node: &DocumentNode, _node_id: NodeId, _context: &mut NodePropertiesContext) -> Vec<LayoutGroup> {
 	let information = WidgetHolder::text_widget("The graph's input is the artwork under the frame layer");
-	let refresh_button = WidgetHolder::new(Widget::TextButton(TextButton {
-		label: "Refresh Input".to_string(),
-		tooltip: "Refresh the artwork under the frame".to_string(),
-		on_update: WidgetCallback::new(|_| DocumentMessage::NodeGraphFrameGenerate.into()),
-		..Default::default()
-	}));
+	let refresh_button = TextButton::new()
+		.label("Refresh Input")
```

Comment:
Can a TextButton's label be part of its `new()` call? That seems inherent to its purpose, not optional.